Lib/test/test_unpack_ex.py

```diff
@@ -256,7 +256,7 @@
     >>> f(**{1: 3}, **{1: 5})
     Traceback (most recent call last):
       ...
-    TypeError: f() keywords must be strings
```

Comment:
I see `f(**{1:2})` is still tested in test_excall.py.

Reply:
Indeed. This is testing two errors at the same time (duplicate keyword and non-string keyword). Which of the two errors you get seems arbitrary to me and this PR changes the error.